server.go

```diff
@@ -1741,8 +1762,20 @@ func (s *server) initialPeerBootstrap(ignore map[autopilot.NodeID]struct{},
 			return
 		}
 
-		// Otherwise, we'll request for the remaining number of peers in
-		// order to reach our target.
+		// We failed to reach our target, so we'll back off a bit to
+		// avoid hammering any of the bootstrap sources.
+		if attempts > 0 {
+			delayTime *= 2
+			if delayTime > backOffCeiling {
+				delayTime = backOffCeiling
+			}
+
+			delayTicker.Stop()
+			delayTicker = time.NewTicker(delayTime)
```

Comment:
might be simpler to keep track of the delay time and then use a `time.After(delayTime)` above, otherwise this ticker may never be stopped